.github/workflows/ci_action.yml

```diff
@@ -11,31 +11,30 @@ env:
 
 jobs:
 
-  check-code-black:
+  check-code-black-isort-flake8:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout branch
         uses: actions/checkout@v2
         with:
           ref: ${{ env.CHECKOUT_BRANCH }}
 
-      - name: Check code compliance with black
-        run: |
-          pip install black[jupyter]
-          black . --check --diff
+      - name: Prepare linters
+        run: pip install black[jupyter] isort flake8 nbqa
 
-  check-code-isort:
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout branch
-        uses: actions/checkout@v2
-        with:
-          ref: ${{ env.CHECKOUT_BRANCH }}
+      - name: Check code compliance with black
+        run: black . --check --diff
 
       - name: Check code compliance with isort
         run: |
-          pip install isort
           isort . --check --diff
+          nbqa isort examples --check --diff
```

Comment:
You could actually do `nbqa isort . --check --diff` because this will recursively find and check all notebooks in all subfolders. Just in case we ever add any other folder with notebooks.

Reply:
hey thanks for trying out nbQA - I'd suggest using `--nbqa-diff` instead of isort's `--diff`, it'll just look a bit better with notebooks

Reply:
@MarcoGorelli  Thanks for the tip!